### ISSUE TITLE
Incorrect codegen

### ISSUE THREAD

**Issue by dustyhorizon** [stephenh/ts-proto#609]
Attempted to generate ts protos using the library with the following options and the codegen seems to have a typo.

Using this repository as a reference, I used the following as parameters `context=true,forceLong=long,esModuleInterop=true,oneof=unions,outputSchema=true,outputTypeRegistry=true,outputServices=generic-definitions,outputServices=grpc-js,outputClientImpl=true` for `integration/grpc-js` and subsequently ran `integration/codegen.sh`

Under `protoMetadata` of the newly generated `simple.ts` exists an error where it produced `references: { '.simple.TestMessage': TestMessage, '.simple.Test': TestClientImpl },` instead of `references: { '.simple.TestMessage': TestMessage, '.simple.Test': TestClient },`

tl;dr Should the generated code say `TestClient` instead of `TestClientImpl` under the `protoMetadata` object?

Thanks!

**Comment by stephenh** [comment]
Ah yeah, thanks for the report @dustyhorizon ...

...just glancing at it, it looks like `grpc-js` client output is slightly different clients, i.e. other outputs do generate separate `TestClient` (interface) and `TestClientImpl` (concrete impl) names, but with grpc-js they are the same:

```
export const TestClient = makeGenericClientConstructor(TestService, 'simple.Test') as unknown as {
  new (address: string, credentials: ChannelCredentials, options?: Partial<ChannelOptions>): TestClient;
  service: typeof TestService;
};
```

So, yeah, this is probably as simple removing the `Impl` from here:

```
  visitServices(fileDesc, sourceInfo, (serviceDesc, sInfo) => {
    if (options.outputClientImpl) {
      references.push(
        code`'.${fileDesc.package}.${serviceDesc.name.replace(/_/g, '.')}': ${serviceDesc.name}ClientImpl`
      );
    }
  });
```

@dustyhorizon if that makes sense / fixes the error, do you mind submitting a PR? Thanks!

@Vilsol as an fyi / in case I'm missing something

**Comment by stephenh** [comment]
Closing as stale
